va/va.go

```diff
@@ -912,12 +912,8 @@ func (va *ValidationAuthorityImpl) validate(
 		ch <- va.checkCAA(ctx, identifier, params)
 	}()
 	go func() {
-		if features.Enabled(features.VAChecksGSB) && !va.isSafeDomain(ctx, baseIdentifier.Value) {
-			ch <- probs.Unauthorized("%q was considered an unsafe domain by a third-party API",
-				baseIdentifier.Value)
-		} else {
-			ch <- nil
-		}
+		ch <- probs.Unauthorized("%q was considered an unsafe domain by a third-party API",
```

Comment:
This is incorrect since it removes the `isSafeDomain` call and just always sends an "unsafe domain" error. I haven't looked at the Travis output, but presumably it's failing for this reason. If there's not a test failure related to this change, we should figure out why and add necessary testing.